docs/modules/getting_started/examples/code/zookeeper.yaml

```
@@ -0,0 +1,11 @@
+---
+apiVersion: zookeeper.stackable.tech/v1alpha1
+kind: ZookeeperCluster
+metadata:
+  name: simple-zk
+spec:
+  version: 3.8.0-stackable0.7.1
+  servers:
+    roleGroups:
+      default:
+        replicas: 2
```

You cannot establish a quorum with just 2 replicas (i think the cluster will still come up, but quorum is mentioned somewhere for readiness so i would use a correct value).
```suggestion
        replicas: 3
```